Writerside/topics/pplib-Build-an-Auto.md

```diff
@@ -510,4 +510,43 @@ class RobotContainer:
 </tab>
 </tabs>
 
+## Create a SendableChooser with certain autos in project
+
+> **Note**
+>
+> This feature is only available in the Java version of PathPlannerLib
+>
+{style="note"}
+
+You can use the buildAutoChooserWithOptionsModifier method to process the 
+autos before they are shown on shuffle board
+
+```java
+public class RobotContainer {
+  private final SendableChooser<Command> autoChooser;
+
+  public RobotContainer() {
+    // ...
+
+    // Check if currently attached to the FMS
+    boolean isCompetition = DriverStation.isFMSAttached();
```

Comment:
Bit of a nitpick but the example here likely wouldn't work since it is possible for code to boot up before connecting to the FMS. Perhaps just setting the boolean to true would be fine?

Reply:
Thank you! Not something I would of thought of...